### PR TITLE
Reduce mac android tests in staging env

### PR DESCRIPTION
**Pull request by keyonghan**
Devicelab Mac/Android tests have been migrated to prod for a while and are running well.
Since we have only one bot available in staging env, this PR leaves only five tests in staging for validation.

Related issue: https://github.com/flutter/flutter/issues/74059

### CODE REVIEW

**Comment by keyonghan** [comment]
A friendly ping.

File: config/devicelab_staging_config.star [hunk 1]
```diff
@@ -207,39 +207,6 @@ def devicelab_staging_prod_config():
         "channels_integration_test",
         "color_filter_and_fade_perf__timeline_summary",
         "complex_layout_scroll_perf__memory",
-        "complex_layout_scroll_perf__timeline_summary",
```
Comment (godofredoc) [inline config/devicelab_staging_config.star:210]: Should we keep a combination of tests with different characteristics? e.g. some using views, some using gallery, etc.

Comment (keyonghan) [reply config/devicelab_staging_config.star:210]: @christopherfujino could you help suggest what combination (around 5-10 tests) to choose?

Comment (jmagman) [reply config/devicelab_staging_config.star:210]: You can move almost all of these benchmarks to Linux/Android.  See https://github.com/flutter/flutter/issues/74522.

**Comment by godofredoc** [comment]
Abandoning this PR for now.

**Pull request closed without merging**